cypress/integration/auth/login.spec.js

```diff
@@ -34,8 +34,9 @@ context('Login', () => {
 
   it('SAAS-T111 SAAS-T81 - should be able to login', () => {
     loginButton().click();
-    cy.get('#okta-signin-submit').should('be.visible');
-    cy.get('#okta-signin-username').type(newUser.email);
+    cy.get('#idp-discovery-username').should('be.visible');
```

Comment:
```suggestion
    cy.get('#idp-discovery-username').should('be.visible').type(newUser.email);
```
You can use this way instead of having 2 lines with the same locator here